tools/doc/html.js

```diff
@@ -36,7 +36,7 @@ marked.setOptions({ renderer });
 
 const docPath = path.resolve(__dirname, '..', '..', 'doc');
 
-const gtocPath = path.join(docPath, 'api', '_toc.md');
+const gtocPath = path.join(docPath, 'api', 'index.md');
 const gtocMD = fs.readFileSync(gtocPath, 'utf8').replace(/^@\/\/.*$/gm, '');
```

Comment:
Sorry, it seems this line also need to be updated so that the new HTML comment from `index.md` could be excluded from all the docs except the `index.html`.

Reply:
I mean the `.replace(/^@\/\/.*$/gm, '')` part.